docs/system-admin-guide/file-storages/file-storage-troubleshooting/README.md

```diff
@@ -0,0 +1,51 @@
+  | Error        | Group does not exist                    | The app was activated on Nextcloud and the group was removed afterwards.<br/>Changes on Nextlcoud: OpenProject group was removed. | Manually add the group in the Nextcloud setup and call it OpenProject. Add the user OpenProject to the group OpenProject. |
+  | Error        | User does not exist                     | After the app was activated on Nextcloud and the user was removed afterwards.  Changes on Nextlcoud: OpenProject user was removed. | Manually add the user in the Nextcloud setup and call that user OpenProject. Add the user OpenProject to the group OpenProject.  <br />Alternatively reinstall the OpenProject integration app on Nextcloud. You will also need to reconfigure the Nextcloud storage. |
+  | Error        | Insufficient privileges                 | OpenProject can not change the user permissions for folders or add folders to the OpenProject folder, because the OpenProject user no longer has access to the folder. | Reinstall the OpenProject integration app on Nextcloud. You will need to reconfigure the Nextcloud storage.   Make sure the OpenProject user is the admin of the OpenProject group and also the admin of the OpenProject folder. |
+  | Error        | Failed to remove or add user from group | A user does not exist in the file storage (Nextcloud or SharePoint).  <br /> A user can not be removed from the OpenProject group due to admin rights.  <br />This may occur when running the sync job. Error only shows in server logs. <br />This error can occur both in Nextcloud and OneDrive/Sharepoint integration. | Ensure that the user exists in the file storage platform. <br />Remove admin rights for that user on the OpenProject group.  <br />If the user is also an admin in the files storage group, he/she/they need to be removed by a file storage platform admin. |
+  | Not allowed  | Outbound request method not allowed     | OpenProject sent wrong requests to the storage.  <br />This error can occur both in Nextcloud and OneDrive/Sharepoint integration. | Report this to [OpenProject community](https://community.openproject.org/projects/openproject/forums) or [support team](https://www.openproject.org/contact/). |
+  | Not found    | Outbound request destination not found  | OpenProject can not reach file storage platform:  <br />- This could be due to Storage provider being down:<br />- DNS problems <br />- Network problems (flaky) <br />- Local networks (Nexctloud specific setting that needs to enabled)  <br />This error can occur both in Nextcloud and OneDrive/Sharepoint integration. | See if you can access the file storage platform from your browser.  <br />For Nextcloud, see if Nextcloud settings are active if in local network. |
```

Comment:
```suggestion
  | Not found    | Outbound request destination not found  | OpenProject can not reach file storage platform.  <br />This could be due to Storage provider being down:<br />- DNS problems <br />- Network problems (flaky network) <br />- Local networks (Nexctloud specific setting that needs to enabled)  | See if you can access the file storage platform from your browser.  <br />For Nextcloud, see if Nextcloud settings are active if in local network. |
```